projects/js-packages/number-formatters/jest.config.cjs

```diff
@@ -0,0 +1,9 @@
+const baseConfig = require( 'jetpack-js-tools/jest/config.base.js' );
+
+module.exports = {
+	...baseConfig,
+	collectCoverageFrom: [
+		'<rootDir>/src/**/*.{js,mjs,cjs,jsx,ts,tsx,mts,cts}',
+		...baseConfig.collectCoverageFrom,
+	],
```

Comment:
I believe this `collectCoverageFrom` is redundant, `baseConfig.collectCoverageFrom` already contains `'<rootDir>/src/**/*.{js,mjs,cjs,jsx,ts,tsx,mts,cts}'`. 🙂

Reply:
Some of my comments may not have come through, are may come later 😬 

I asked a question from the editor, whether this file is needed at all.

Reply:
I think some config file is needed to load the basic configuration, but many are as simple as
```
const baseConfig = require( 'jetpack-js-tools/jest/config.base.js' );

module.exports = {
        ...baseConfig,
};
```

Reply:
got it. thanks